src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -453,6 +453,39 @@ type Todo @model {
 If you create a new Todo and don't supply a `content` input, Amplify will ensure that `My new Todo` is auto populated as a value. 
 When `@default` is applied, non-null assertions using `!` are disregarded. For example, `String!` is treated the same as `String`.
 
+## Subscriptions Runtime Filtering
+
+You can filter the subscriptions server-side by passing a filter expression.
+
+```graphql
+type Task @model {
+  title: String!
+  description: String
+  type: String
+  priority: Int
+}
+```
+
+If you want to subscribe to tasks of type `Security` and priority greater than `5`, you can set the `filter` argument accordingly.
```

Comment:
You can filter the subscriptions server-side by passing a `filter` expression. For example: .....